.github/workflows/verify-azure-issue.yml

```diff
@@ -0,0 +1,41 @@
+name: "Verify Linked Task"
+
+on: [pull_request]
+
+# For details refer to https://github.com/tim-actions/commit-message-checker-with-regex
+jobs:
+  check-pr:
+    if: ${{ !startsWith(github.head_ref, 'dependabot') }}
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR Commits
+        id: 'get-pr-commits'
+        uses: tim-actions/get-pr-commits@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check Subject Line Length
+        uses: tim-actions/commit-message-checker-with-regex@v0.3.1
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
+          pattern: '^.{0,128}(\n.*)*$'
+          error: 'Subject too long (max 75)'
```

Comment:
max 75 is now wrong. I would put the recomendation in the error too:

'Subject too long, recomended is under 50 (max 128)'

Reply:
fixed